### ISSUE TITLE
Implement testing for various packages

### ISSUE THREAD

**Issue by moonmeister** [gatsby-uc/plugins#70]
Most of the packages here don't have testing. I don't have time to implement but if anyone does it'd be super helpful in giving more confidence around package updates and deploys. 

- [ ] gatsby-plugin-fastify
- [ ] gatsby-plugin-relative-ci
- [ ] gatsby-plugin-reading-time
- [ ] gatsby-plugin-ackee-tracker

**Comment by laneparton** [comment]
@moonmeister Do you know any example repos or packages that might serve as a good reference?

**Comment by moonmeister** [comment]
Yeah. I'd go checkout packages in the Gatsby repo. To keep things easy we'll probably use just and and the same setup patterns. I've worked on "gatsby-plugin-manifest" manifest but there are lots of plugins with tests.

**Comment by moonmeister** [comment]
https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-manifest/src/__tests__

https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-sitemap/src/__tests__

**Comment by laneparton** [comment]
@moonmeister Thanks Alex! This is something I've been looking to get some experience with. I'll start poking around tonight 😄 

If I have any specific questions - I might ping you on Discord!
